src/server.ts

```diff
@@ -511,7 +511,9 @@ connection.onDidChangeConfiguration((change) => {
 
   schemaConfigurationSettings = [];
 
-  if (settings.editor?.tabSize) {
+  if (settings['[yaml]'] && settings['[yaml]']['editor.tabSize']) {
+    indentation = ' '.repeat(settings['[yaml]']['editor.tabSize']);
```

Comment:
Should we document this somewhere just so people know that if their editor supports yaml specific tabSizes then it takes precedence over the regular editor.tabSize

Reply:
Hm, not sure, do we have some description of settings that we use ?

Reply:
We only really have https://github.com/redhat-developer/yaml-language-server#language-server-settings but since the number of spaces is grabbed from editor.tabSize we should probably add a little writeup there. I can do it in a separate PR if you want

Reply:
Yes, that helps, pleas include also `proxy` and `proxyStrictSSL` settings usage